examples/updatecli.generic/githubRelease.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
How about taking back the top comment block of the old template? Or is this content stored elsewhere?

Reply:
the former top comment block was "telling a story" about the former example (including the conditions and targets that are not supported by githubrelease).

The goal of the change is to provide a generic example specific to githubrelease.

Also, https://www.updatecli.io/docs/resources/github_release/#_example has plenty of the same content and explanation :)